suite/fuzz/fuzz_disasm.c

```diff
@@ -332,6 +332,18 @@ static struct platform platforms[] = {
         "M680X_M6808",
         "hcs08"
     },
+    {
+        //item 33
```

Comment:
item 44, and another string to it, the same that is used in stool, ie "risc32" again